src/firefly/html/demo/ffapi-highlevel-test.html

```diff
@@ -0,0 +1,116 @@
+    <meta http-equiv="Cache-Control" content="no-cache">
+    <meta http-equiv="Pragma" content="no-cache">
+    <meta http-equiv="Expires" content="0">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <title>Demo of Firefly Tools</title>
+</head>
+
+<body>
+
+
+<div style="width: 500px; padding: 20px 0 0 20px;">
+    <br>
+    This page demos new api
+    <br>
+</div>
+
+<div id="imageViewHere" style="width: 550px; height: 550px; padding: 10px 5px 5px 5px;"></div>
+<div id="old1" style="width: 300px; height: 300px; padding: 10px 5px 5px 5px;"></div>
+<div id="old2" style="width: 300px; height: 300px; padding: 10px 5px 5px 5px;"></div>
+
+
+<script type="text/javascript">
+    {
+        onFireflyLoaded= function(firefly) {
+            firefly.setGlobalImageDef({
+                ZoomType  : 'TO_WIDTH'
+            } );
+
+            firefly.debug= true;
+            var req= {
+                Type      : 'SERVICE',
+                plotGroupId : 'myGroup',
+                Service   : 'TWOMASS',
+                Title     : '2mass from service',
+                GridOn     : 'true',
+                SurveyKey  : 'k',
+                WorldPt    : '10.68479;41.26906;EQ_J2000',
+                SizeInDeg  : '.12',
+                AllowImageSelection : true
+            };
+
+            var req2= {
+                Type      : 'SERVICE',
+                plotGroupId : 'myGroup',
+                Service  : 'WISE',
+                Title     : 'Wise',
+                GridOn     : 'true',
+                SurveyKey  : 'Atlas',
+                SurveyKeyBand  : '2',
+                WorldPt    : '10.68479;41.26906;EQ_J2000',
+                SizeInDeg  : '.12',
+                RangeValues : firefly.util.image.RangeValues.serializeSimple('Sigma',-2,8,'Linear'),
+                AllowImageSelection : true };
+
+
+
+            var threeC= [
+                {
+                    type      : 'SERVICE',
+                    plotGroupId : 'myGroup',
+                    Service   : 'TWOMASS',
+                    Title     : '3 color',
+                    SurveyKey  : 'j',
+                    WorldPt    : '10.68479;41.26906;EQ_J2000',
+                    SizeInDeg  : '.12'
+                },
+                {
+                    Type      : 'SERVICE',
+                    PLOTgroupId : 'myGroup',
+                    Service   : 'TWOMASS',
+                    Title     : '3 color',
+                    SurveyKey  : 'h',
+                    WorldPt    : '10.68479;41.26906;EQ_J2000',
+                    SizeInDeg  : '.12'
+                },
+                {
+                    Type      : 'SERVICE',
+                    plotGroupId : 'myGroup',
+                    Service   : 'TWOMASS',
+                    title     : '3 color',
+                    SurveyKey  : 'k',
+                    WorldPt    : '10.68479;41.26906;EQ_J2000',
+                    SizeInDeg  : '.12'
+                }
+            ];
+
```

Comment:
I could expose WebPlotRequest which has many helper functions.  then it would be either a WebPlotRequest object or a object literal.  It would work, maybe more confusing.